src/components/blocks/layout/Header.tsx

```diff
@@ -51,17 +98,38 @@ export const Header = ({ data }: { data: PageBlocksHeader }) => {
 			)}
 		>
 			<div className='flex max-w-7xl justify-between items-center w-full p-4'>
```

Comment:
You can change `p-4` to `py-4 and px-8` to add padding to the outer edge to make sure it doesn't touch the edge of the screen